src/subcommands/subcommand.go

```diff
@@ -57,6 +60,13 @@ func CommandReadRun() {
 
 	setupSemantifly()
 
+	ctx, conn, err := setupDBConn()
```

Comment:
Why did you do this here if you are also doing it for each subcommand in server.go?

Reply:
Yeah I'm doing it twice. I'll send this connection to the server